--- origsrc/crafty-21.6/chess.h	2007-10-29 12:58:07.000000000 -0500
+++ src/crafty-21.6/chess.h	2007-12-25 10:11:59.937500000 -0600
@@ -118,6 +118,9 @@
 #  if !defined(RCDIR)
 #    define       RCDIR        "."
 #  endif
+#  if !defined(HELPDIR)
+#    define       HELPDIR        "."
+#  endif
 #  if !defined(NOEGTB)
 #    define     EGTB_CACHE_DEFAULT               1024*1024
 #  endif
--- origsrc/crafty-21.6/option.c	2007-10-29 12:58:07.000000000 -0500
+++ src/crafty-21.6/option.c	2007-12-25 10:12:16.859375000 -0600
@@ -1333,7 +1333,7 @@
     char *readstat = (char *) -1;
     int lines = 0;
 
-    helpfile = fopen("crafty.hlp", "r");
+    helpfile = fopen(HELPDIR "/crafty.hlp", "r");
     if (!helpfile) {
       printf("ERROR.  Unable to open \"crafty.hlp\" -- help unavailable\n");
       return (1);
